static/app/components/events/viewHierarchy/utils.tsx

```diff
@@ -0,0 +1,120 @@
+import {useLayoutEffect, useState} from 'react';
```

Comment:
Aside from the introduction of `getDeepestNodeAtPoint`, the rest of these are just utils moved out of the `wireframe.tsx` file